vkcs/compute/resource_volume_attach_test.go

```diff
@@ -25,6 +25,47 @@ func TestAccComputeVolumeAttach_basic(t *testing.T) {
 				Config: acctest.AccTestRenderConfig(testAccComputeVolumeAttachBasic),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckComputeVolumeAttachExists("vkcs_compute_volume_attach.va_1", &va),
+					resource.TestCheckResourceAttr("vkcs_blockstorage_volume.volume_1", "metadata.key1", "val0"),
```

Comment:
Why not use the same `testAccCheckBlockStorageVolumeMetadata` here?

Reply:
use it for all_metadata. Unfortunately, this cannot be done for the metadata field because it does not match
the cloud metadata